### ISSUE TITLE
Lien de sortie Etudes de cas

### ISSUE THREAD

**Issue by xgorse** [Elao/elao_#257]
A la fin d"une étude de cas dans les blocs Tribu et  Autre Etudes rajouter des liens sur les pages de liste respectives

https://www.elao.com/la-tribu
https://www.elao.com/etudes-de-cas

![image](https://user-images.githubusercontent.com/117971/114042889-1d53c180-9886-11eb-8916-3dc493581a00.png)

**Comment by ameliedefrance** [comment]
@xgorse @cmozzati ça me semble résolu, je me trompe ? Exemple : https://www.elao.com/etudes-de-cas/musique-music

**Comment by cmozzati** [comment]
@ameliedefrance @sludovicdelys non, le propos de @xgorse est de mettre des liens sur les titres "La tribu" / "D'autres études de cas"
